app/src/main/java/bubbleheads/buy_cook/CategoriesAdapter.java

```diff
@@ -1,36 +1,30 @@
 package bubbleheads.buy_cook;
 
 import android.support.v7.widget.RecyclerView;
-import android.util.Log;
+import android.support.v7.widget.RecyclerView.Adapter;
+import android.support.v7.widget.RecyclerView.ViewHolder;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.BaseAdapter;
 import android.widget.TextView;
-import android.support.v7.widget.RecyclerView.Adapter;
-import android.support.v7.widget.RecyclerView.ViewHolder;
+import android.widget.Toast;
 
 import bubbleheads.buy_cook.CategoriesAdapter.MyViewHolder;
 
-import org.w3c.dom.Text;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static com.google.android.gms.internal.zzs.TAG;
-
 public class CategoriesAdapter extends Adapter<MyViewHolder> {
     private final CategoriesFragment categoriesFragment;
 
     protected static class MyViewHolder extends ViewHolder {
 
         private final TextView text;
         private final View view;
+        private final RecyclerView rv;
```

Comment:
inconsistent name. App uses full names everywhere but here you have shortened one. Change it to recycleView